src/components/ActionButton/ActionButton.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { FC } from 'react';
+import classes from '../ExpandedNode/ExpandedNode.module.scss';
+import indexClasses from '../../styles/index.module.scss';
+import withConfirmationPopup from '../withPopup/withConfirmationPopup';
```

Comment:
According to the style guide these should be called `styles` and `indexStyles` and use an absolute import